test/list_differ_test.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:observable/observable.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('$ListDiffer', () {
```

Comment:
I've seen some tests just supply the type. `group(ListDiffer, () {

I don't feel strongly about it, but something to consider.

Reply:
Ack. I'll keep it consistent for now. If I change this I'll do it as part of a larger change that also updates the other tests.